internal/engine/planner/planner.go

```diff
@@ -101,8 +140,16 @@ func (p *PolicyPlanResult) Add(filter *qpN, effect effectv1.Effect) {
 	}
 }
 
+func (p *PolicyPlanResult) DenyEmpty() bool {
+	return len(p.DenyFilter) == 0
+}
+
+func (p *PolicyPlanResult) AllowEmpty() bool {
```

Comment:
When happening across this function above, my immediate assumption was that it would be allowing something rather than just determining if the allow filters are empty. Perhaps `AllowIsEmpty` or something along those terms would be more clear? Same for `DenyIsEmpty`?